content/en/docs/contribute/participate/roles-and-responsibilties.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Roles and responsibilities
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+Anyone can contribute to Kubernetes. As your contributions to SIG Docs grow, you can apply for different levels of membership in the community. These roles allow you to take on more responsibility within the community. Each role requires more time and commitment. The roles are:
+
+- Anyone: regular contributors to the Kubernetes documentation
+- Members: can assign and triage issues and provide non-binding review on pull requests
+- Reviewers: can lead reviews on documentation pull requests and can vouch for a change's quality
+- Approvers: can lead reviews on documentation and  merge changes
+
+<!-- body -->
+
+## Anyone
+
+Anyone with a GitHub account can contribute to Kubernetes. SIG Docs welcomes all new contributors!
+
+Anyone can:
+
+- Open an issue in any [Kubernetes](https://github.com/kubernetes/) repository, including [`kubernetes/website`](https://github.com/kubernetes/website)
+- Give non-binding feedback on a pull request
+- Localize existing content
```

Comment:
What does `Localize existing content` mean?

Reply:
Localize existing content = participate/contribute to a localization